doc/manual/de/config/customizing.rst

```diff
@@ -0,0 +1,273 @@
+werden die verschiedenen Möglichkeiten - sortiert nach Komplexitätsgrad - im Detail beschrieben.
+
+===   =============================    ================    ===================================================
+\     Anpassung                        Veränderung          Beschreibung
+===   =============================    ================    ===================================================
+1.    Eigene CSS-Regeln                Optisch             Überschreiben der vorhandenen CSS-Regeln eines Designs durch das Laden von zusätzlichen CSS-Dateien
+2.    Eigene Icons einbinden           Optisch             Neben den mitgelieferten Icons können eigenen Icons über die Konfigurationsdatei hinzugefügt werden
+3.    Widgets durch ``class``          Optisch             Viele Widgets können individualisiert werden durch Hinzufügen des ``class`` Attributs und dazu passenden CSS-Regeln
+4.    Eigenes Design schreiben         Optisch             Sollten zusätzliche CSS-Regeln nicht ausreichen, kann ein neuen Design erstellt werden
+5.    Eigene Widgets schreiben         Inhaltlich          Hinzufügen neuer Widgets durch Plugins.
+6.    Eigene Struktur schreiben        Inhaltlich          Die Umwandlung der Konfigurationsdateien in HTML-Code, kann durch erstellen einer neuen Struktur komplett verändert werden
+===   =============================    ================    ===================================================
+
+Vorhandenes Design verändern
+----------------------------
+
+Die vorhandenen Designs können durch eigene CSS-Regeln angepasst werden. Im Meta-Bereich der Konfigurationsdatei
+können dazu Pfade zu CSS-Dateien angegben werden, die zusätzlich zu den CSS-Dateien des CometVisu-Designs geladen werden.
+
+.. code-block:: xml
+    :caption: Auszug einer Konfigurationsdatei, die 2 zusätzliche CSS-Dateien lädt:
+
+    <pages... design="metal">
+        <meta>
+            <files>
+                <file type="css">resource/my-custom-style.css</file>
+                <file type="css">resource/my-other-custom-style.css</file>
+                ...
+            </files>
+            ...
+        </meta>
+        ...
+    </pages>
+
+.. HINT::
+    In früheren CometVisu-Versionen (<= 0.10.x) mussten die CSS-Regeln in die ``custom.css`` Datei des jeweiligen
+    Designs eingetragen werden. Auch wenn dies weiterhin funktioniert, wird empfohlen diese Regeln in eine neue Datei zu kopieren
+    und über den hier beschriebenen Weg einzubinden.
+
+
+Eigene Icons einbinden
+----------------------
+
+Mit dieser Option wird der Name des Icons definiert, welches sich unter dem in uri angegebenen Verzeichnis befindet.
+Auf die so definierten Icons kann dann im weiteren Verlauf über den einfacher zu merkenden Namen zugegriffen werden.
+Die Verzeichnisangabe ist im Beispiel relativ zur CV installation.
+Hier wurden vorher die Icons in einem eigenen Unterverzeichnis abgelegt.
```

Comment:
I'd change the example to use the "best practice" directory `config/media/` for those additional (media) files